test/kwtool_web/controllers/keyword_controller_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule KwtoolWeb.KeywordControllerTest do
+  use KwtoolWeb.ConnCase
+
+  describe "get index/2" do
+    test "returns the keywords page when given a logged-in user", %{conn: conn} do
+      created_user = insert(:user)
```

Comment:
Let's create keywords for this user and another user then add an assertion for those keywords.

Reply:
yes @hanam1ni ! noted, however, I will add it to the backend PR. As this PR is for the frontend only, that's why I don't put the code for querying keywords of a specific user.

Reply:
Noted 👌